src/testing/jest/jest-27-and-under/jest-config.ts

```diff
@@ -81,8 +82,10 @@ export function buildJestArgv(config: d.ValidatedConfig): Config.Argv {
     } catch (e) {}
   }
 
-  if (typeof jestArgv.ci === 'string') {
-    jestArgv.ci = jestArgv.ci === 'true' || jestArgv.ci === '';
+  for (const flag of BOOLEAN_CLI_FLAGS) {
+    if (typeof jestArgv[flag] === 'string') {
+      jestArgv[flag] = jestArgv[flag] === 'true';
```

Comment:
We breaking anything by removing  the `jesArgv[flag] === ''` check?

Reply:
You know, I dont _think_ so, but maybe to be sure I'll leave the existing special-casing here for the `ci` flag just in case